tests/test_pillow.py

```diff
@@ -255,6 +255,23 @@ def test_gif_gray(test_images, tmp_path):
     )
 
 
+def test_gif_fps_error(test_images, tmp_path):
+    im = iio.v3.imread(
+        test_images / "newtonscradle.gif",
+        plugin="pillow",
+        mode="L",
+    )
+
+    with pytest.raises(TypeError, match="The keyword, `fps`"):
```

Comment:
```suggestion
    with pytest.raises(TypeError):
```

Sorry, I forgot to make a suggestion here. We typically don't use the `match=` part of `pytest.raises`. It has its uses, but for us it is enough if the test "fails" with the correct exception.